python/ray/dag/compiled_dag_node.py

```diff
@@ -737,7 +741,7 @@ def __init__(
         self._max_finished_execution_index: int = -1
         self._result_buffer: Dict[int, Any] = {}
 
-        def _get_creator_or_proxy_actor() -> "ray.actor.ActorHandle":
+        def _get_proxy_actor() -> "ray.actor.ActorHandle":
             """
             Get the creator actor or proxy actor handle of the DAG.
```

Comment:
update the comments

Reply:
actually I don't think it needs docstring. just removed